go/chat/inboxsource.go

```diff
@@ -219,77 +242,85 @@ func (s *RemoteInboxSource) localizeConversationsPipeline(ctx context.Context, c
 }
 
 func (s *RemoteInboxSource) localizeConversation(
-	ctx context.Context, conversationRemote chat1.Conversation) (
-	conversationLocal chat1.ConversationLocal, err error) {
+	ctx context.Context, conversationRemote chat1.Conversation, identifyBehavior keybase1.TLFIdentifyBehavior) (
+	conversationLocalWithBreaks chat1.ConversationLocalWithBreaks, err error) {
 
 	ctx, uimap := utils.GetUserInfoMapper(ctx, s.G())
 
-	conversationLocal.Info = chat1.ConversationInfoLocal{
+	conversationLocalWithBreaks.ConversationLocal.Info = chat1.ConversationInfoLocal{
```

Comment:
This is also super verbose, if we could reduce the length of these names it would be great.